MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
+include truman/_version.py
```

Comment:
I feel like we're serious mofos now that we have a manifest file.

Reply:
lol versioneer made it for me 🥵